internal/hmyapi/blockchain.go

```diff
@@ -64,7 +64,8 @@ func (s *PublicBlockChainAPI) GetBlockByHash(ctx context.Context, blockHash comm
 }
 
 // GetCode returns the code stored at the given address in the state for the given block number.
-func (s *PublicBlockChainAPI) GetCode(ctx context.Context, address common.Address, blockNr rpc.BlockNumber) (hexutil.Bytes, error) {
+func (s *PublicBlockChainAPI) GetCode(ctx context.Context, adr string, blockNr rpc.BlockNumber) (hexutil.Bytes, error) {
```

Comment:
ubernit: haven't seen `adr` as abbreviation for address, how about `addr`.

Reply:
fixed